CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+Copyright 2018 Yahoo Inc.
```

Comment:
Should be 
https://github.com/screwdriver-cd/screwdriver/blob/master/CONTRIBUTING.md

Reply:
Updated